xtable-utilities/src/main/java/org/apache/xtable/utilities/RunSync.java

```diff
@@ -272,6 +307,19 @@ public static class Table {
       String partitionSpec;
       String namespace;
     }
+
+    @Data
+    public static class Catalog {
```

Comment:
@the-other-tim-brown User needs to populate the catalog configs in the following format, will finish up the tests if this experience is okay.